### ISSUE TITLE
Bug in glazing coating sides

### ISSUE THREAD

**Issue by Mathadon** [open-ideas/IDEAS#464]
When changing port a and port b we forgot to switch the sides of the coatings for the glazing. This did not affect the results significantly since two errors compensated each other. However the code should be fixed. I'll make a pull request.

**Comment by rubenbaetens** [comment]
Could you tell me what it means in `IDEAS.Buildings.Data.Glazing` ?

The current representation is:
```
    mats={Materials.Glass(d=0.006, epsLw_b=0.110),
                Materials.Argon(d=0.016),
                Materials.Glass(d=0.006, epsLw_a=0.10)},
```
for low-e coatings inside the cavity. Is this correct or what should it be ?

**Comment by Mathadon** [comment]
It should be 
```
mats = {Materials.Glass(d=0.006, epsLw_a=0.110),
        Materials.Argon(d=0.016),
        Materials.Glass(d=0.006, epsLw_b=0.10)},
```
since the low - e coatings are inside the cavity and `port_a` is connected to the last layer.

**Comment by rubenbaetens** [comment]
I'm waiting :)

**Comment by Mathadon** [comment]
Working on it!

**Comment by rubenbaetens** [comment]
Solved in #467.
